src/components/user-profile-page/_user-profile-page.scss

```diff
@@ -0,0 +1,314 @@
+
+    img {
+      display: block;
+      margin: auto;
+      padding: 0 2.1rem;
+      width: 85%;
+    }
+  }
+
+  &__header__title {
+    font-size: 4rem;
+    padding: 7rem 3rem 0rem;
+    text-align: center;
+  }
+
+  &__header__description {
+    font-size: 2.5rem;
+    padding: 2rem 5rem 8rem;
+    text-align: center;
+  }
+
+  &__header__arrow-button {
+    background: #383d3f url("/components/user-profile-page/images/arrow.svg") no-repeat center center;
+    border-radius: 50%;
+    cursor: pointer;
+    height: 60px;
+    left: 0;
+    margin: 0 auto;
+    position: absolute;
+    right: 0;
+    text-align: center;
+    transform: rotate(90deg);
+    transition: transform 250ms ease-out;
+    width: 60px;
+  }
+
+  &__form {
+    font-size: 1.6rem;
+    padding: 5rem 2rem 0rem;
+
+    .input-group {
+      margin: 0 0 2rem 0;
+    }
+
+    label {
+      display: block;
+    }
+
+    input {
+      background-color: #f5f8f9;
+      border: 1px solid #cad8de;
+      border-radius: 0.5rem;
+      margin-top: 1rem;
+      padding: 1rem;
+      width: 100%;
+
+      &:invalid {
+        color: #ff9200;
+      }
+    }
+
+    .radio-input {
+      align-items: center;
+      display: inline-flex;
+      margin-right: 2.4rem;
+
+      :focus,
+      :hover {
+        border: 1px solid #b1bfcd;
+      }
+
+      span {
+        &:hover {
+          border: none;
+        }
+      }
+
+      input[type="radio"] {
+        appearance: none;
+        background-position: center center;
+        background-repeat: no-repeat;
+        border-radius: 50%;
+        cursor: pointer;
+        height: 5.2rem;
+        margin-right: 2rem;
+        width: 5.2rem;
+
+        &:checked {
+          background: #b1bfcd;
+          background-position: center center;
+          background-repeat: no-repeat;
+        }
+      }
+    }
+
+    input[type="radio"][name="gender"][value="male"] {
+      background-image: url("/components/user-profile-page/images/mars-symbol.svg");
+
+      &:checked {
+        background-image: url("/components/user-profile-page/images/mars-symbol--white.svg");
+      }
+    }
+
+    input[type="radio"][name="gender"][value="female"] {
+      background-image: url("/components/user-profile-page/images/venus-symbol.svg");
+
+      &:checked {
+        background-image: url("/components/user-profile-page/images/venus-symbol--white.svg");
+      }
+    }
+
+    input[type="radio"][name="membership"] {
+      background-image: url("/components/user-profile-page/images/card.svg");
+
+      &:checked {
+        background-image: url("/components/user-profile-page/images/card--white.svg");
+      }
+    }
+
+    &__buttons {
+      margin-top: 2rem;
+
+      button {
+        border: none;
+        border-radius: 0.5rem;
+        cursor: pointer;
+        margin-bottom: 2rem;
+        padding: 2rem;
+        text-transform: uppercase;
+        width: 100%;
+      }
+
+      &-cancel {
+        background-color: #f5f8f9;
```

Comment:
@3aPretorius I think the styling for the cancel button should be moved into the base button styling instead.